Dockerfile.circleci

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu
+
+RUN mkdir -p /opt/penrose
+COPY . /opt/penrose/
+WORKDIR /opt/penrose
+RUN apt-get update && apt-get install -y \
+  ca-certificates \
+  libgmp-dev
+
+ENTRYPOINT ["./penrose"]
+EXPOSE 9160
+CMD ["--domain=0.0.0.0","src/sty/venn.sty","src/set-theory-domain/setTheory.dsl"]
```

Comment:
We should change the CLI to use the remote version of the files instead in the near future

Reply:
Maybe allow omitting the domain files as arguments altogether? We can put the burden on the frontend to send them over.